MonoGame.Framework/Graphics/GraphicsDevice.OpenGL.cs

```diff
@@ -118,32 +122,43 @@ private void PlatformSetup()
                     0;
 
                 var samples = 0;
-                if (Game.Instance.graphicsDeviceManager.PreferMultiSampling)
+                if (Game.Instance != null)
                 {
-                    // Use a default of 4x samples if PreferMultiSampling is enabled
-                    // without explicitly setting the desired MultiSampleCount.
-                    if (PresentationParameters.MultiSampleCount == 0)
+                    if (Game.Instance.graphicsDeviceManager.PreferMultiSampling)
                     {
-                        PresentationParameters.MultiSampleCount = 4;
-                    }
+                        // Use a default of 4x samples if PreferMultiSampling is enabled
+                        // without explicitly setting the desired MultiSampleCount.
+                        if (PresentationParameters.MultiSampleCount == 0)
+                        {
+                            PresentationParameters.MultiSampleCount = 4;
```

Comment:
This logic should be done within `GraphicsDeviceManager` if that is the case.  At this point the code should just obey the `MultiSampleCount` setting.